napari/components/viewer_model.py

```diff
@@ -571,6 +571,99 @@ def add_image(
                 layer_list.append(layer)
             return layer_list
 
+    def add_text(
+        self,
+        data=None,
+        *,
+        rotation=0,
+        text_color='black',
+        font_size=12,
+        font='OpenSans',
+        anchor_x='center',
+        anchor_y='center',
+        render_method='cpu',
+        name=None,
+        metadata=None,
+        scale=None,
+        translate=None,
+        opacity=1,
+        blending='translucent',
+        visible=True,
+    ):
+        """Add a text layer to the layers list.
+
+        Parameters
+        ----------
+        data : tuple (coords, text)
+            coords contains text coordinates for N points in D dimensions.
+            text contains a list of the strings to be displayed as text.
+        rotation : float
+            Angle of the text elements
+        text_color : str
+            The color of the text font.
+        font_size : float
+            Size of the text font in points.
+        font : str
+            Text font. OpenSans is the default.
+        anchor_x : str
```

Comment:
Can anchor be a 2-tuple of str `('center', 'left')`. Can you also list here what are the acceptable string values (if there are only a few)